app/(routes)/organizers/page.tsx

```diff
@@ -1,193 +1,46 @@
-    imageUrl:
-      "https://media.licdn.com/dms/image/D4E03AQH2PwcKtDoGAg/profile-displayphoto-shrink_200_200/0/1670726619198?e=1695254400&v=beta&t=pbrUCiZcFNtqf6pavxPvcCiYmAezxWbYABuOCnTrymQ"
-  },
-  {
-    name: "Shirly Spikes",
-    role: "Volunteer",
-    linkedinUrl: "https://www.linkedin.com/in/shirly-spikes-725a413b/",
-    // teamLead: 'Team Lead:',
-    team: 'Features',
-    imageUrl:
-      "https://media.licdn.com/dms/image/C4D03AQF3qK7KUlfxHQ/profile-displayphoto-shrink_200_200/0/1609945840249?e=1695254400&v=beta&t=0WlFasb-0atWKFhlY4_XqyYP2W7CE1zAuE7z0WnK2h0"
-  },
-  {
-    name: "Sai Dutt",
-    role: "Volunteer",
-    linkedinUrl: "https://www.linkedin.com/in/dutt-sai/",
-    // teamLead: 'Team Lead:',
-    team: '',
-    imageUrl:
-      "https://media.discordapp.net/attachments/1130964544706056303/1133814348565975050/PXL_20230101_190254075_2.jpg?width=756&height=920"
-  },
-  {
-    name: "Sean Daly",
-    role: "Volunteer",
-    linkedinUrl: "https://www.linkedin.com/in/seancdaly/",
-    // teamLead: 'Team Lead:',
-    team: '',
-    imageUrl:
-      "https://media.licdn.com/dms/image/D5603AQEo8O0zmNabkA/profile-displayphoto-shrink_200_200/0/1689400196234?e=1695254400&v=beta&t=CIQJ7UX5cwpPA0mMQ7WyzsQ3TMKzbZ00CVNX59I--Z8"
-  },
-  {
-    name: "Claudio Olmedo",
-    role: "Volunteer",
-    linkedinUrl: "https://www.linkedin.com/in/claudioolmedo-com/",
-    // teamLead: 'Team Lead:',
-    team: '',
-    imageUrl:
-      "https://media.licdn.com/dms/image/D4E03AQE6q5fDJj5zcA/profile-displayphoto-shrink_200_200/0/1685118707659?e=1695254400&v=beta&t=MTVN3lH51OFLr27bpGGep401OW3RffbA8vNeANqZGZY"
-  },
-
-
-
-];
-
-export default function Organizers() {
-  const organizers = team.filter(person => person.role === 'Co-Organizer');
-  const volunteers = team.filter(person => person.role !== 'Co-Organizer');
-
+import { Client } from "@notionhq/client";
+
+const notion = new Client({ auth: process.env.NOTION_API_KEY });
+
+// this function fetches data from The Notion Database(Table). 
+const fetchDataFromNotion = async () => {
+  const databaseId = process.env.NOTION_API_DATABASE;
+  const response = await notion.databases.query({ database_id: databaseId });
+
+  // we map through the array of object to pull out the properties that we need for rendering.
+  const teamResults = response.results.map((page) => {
+    return {
+      id: page.id,
+      name: page.properties.Name.title[0]?.plain_text,
+      role: page.properties.Role.multi_select[0].name,
+      // team: page.properties.Teams.rich_text[0]?.plain_text,
+      imageUrl: page.properties.Image.files[0]?.file.url,
+      linkedInUrl: page.properties.LinkedIn.rich_text[0]?.plain_text,
+      notionId: page.properties.NotionId.rich_text[0]?.plain_text
+    };
+  });
+
+  //this function sorts the array by NotionId property that we set in the Notion table
+  teamResults.sort((a, b) => {
+    return parseInt(a.notionId) - parseInt(b.notionId);
+  });
+  return teamResults;
+};
```

Comment:
very neat! can we mode this notion code to its own file? Maybe along with the other services  `service/notion/NotionService`. We do this separation of concerns so that the code can be reused by other parts of the applications.

Reply:
moved the function to the app/service/notion/NotionService as requested.